src/context.cpp

```diff
@@ -87,8 +87,9 @@ namespace Sass {
 
   {
 
-    // add cwd to include paths
-    include_paths.push_back(CWD);
+    // Sass 3.4: The current working directory will no longer be placed onto the Sass load path by default.
```

Comment:
You're right that this is 3.5 now, but I was thinking for ease of backporting changes to 3.4 while they are still both being maintained. I guess it would need to be a plain definition of `SASS_LANGUAGE_3_4`